README.rst

```diff
@@ -1,42 +1,64 @@
-Course Discovery plugin for `Tutor <https://docs.tutor.edly.io>`_
-=====================================================================
+Course Discovery plugin for `Tutor`_
+====================================
 
-This is a plugin for `Tutor <https://docs.tutor.edly.io>`_ that integrates the `Course Discovery <https://github.com/edx/course-discovery/>`__ application in an Open edX platform. it is useful for integration with, for example, `Ecommerce <https://github.com/edx/ecommerce>`__ or an external course catalog.
+This is a plugin for `Tutor`_ that integrates the `Course Discovery`_ application in an Open edX platform.
+it is useful for integration with, for example, `Ecommerce`_ or an external course catalog.
+
+
+.. _Tutor: https://docs.tutor.edly.io
+.. _Course Discovery: https://github.com/edx/course-discovery/
+.. _Ecommerce: https://github.com/edx/ecommerce
 
 Installation
 ------------
 
-This plugin requires tutor>=3.6.0. If you have installed tutor from a pre-compiled binary, it already comes bundled with the discovery plugin. Otherwise::
+This plugin requires ``tutor>=3.6.0``. If you have installed tutor from a pre-compiled binary,
+it already comes bundled with the discovery plugin. Otherwise:
+
+.. code-block:: bash
+
 
     tutor plugins install discovery
 
-Then, to enable this plugin, run::
+Then, to enable this plugin, run:
+
+.. code-block:: bash
 
     tutor plugins enable discovery
 
-Restart and initialize your platform with::
+Restart and initialize your platform with:
+
+.. code-block:: bash
 
-    tutor local launch
+    tutor dev|local|k8s launch
 
 Operations
 ----------
 
 Creating a user
 ~~~~~~~~~~~~~~~
 
-The discovery user interface will be available at http://discovery.local.edly.io for a local test instance, and at ``DISCOVERY_HOST`` (by default: http(s)://discovery.<your lms host>) in production. In order to run commands from the UI, a user must be created::
+The discovery user interface will be available at http://discovery.local.edly.io for a local test instance,
+and at ``DISCOVERY_HOST`` (by default: http(s)://discovery.<your lms host>) in production. To run
+commands from the UI, a user must be created:
+
+.. code-block:: bash
 
     tutor local run discovery ./manage.py createsuperuser
```

Comment:
why `local` is being colored as red in every command after the changes? Please keep it black.

Reply:
That's because of syntax highlighting for bash codes, in Github.